docs/first_semantic_game/01.project_setup.rst

```diff
@@ -5,20 +5,16 @@ Creating a New Godot Project
 ----------------------------
 
 1. Launch Godot
-2. Navigate to "New Project" in the Project Manager.
-3. Define a project name and designate a location on the system for project files. Create a dedicated folder for this purpose.
-   The compatibility renderer works fine for our purposes.
-
-For more information on using C# with Godot see https://docs.godotengine.org/en/stable/tutorials/scripting/c_sharp/c_sharp_basics.html
+2. Click ``+ New Project`` in the Project Manager and create a new project.
 
 Hello World Script
 ------------------
 
-First, let's create a simple C# script which will trigger Godot to initialize
+First, let's create a simple C# script. Doing so will cause Godot to initialize
 a dotnet project.
 
 1. **Create a Node and Attach the Script**:
-    - Create a new 2D scence with a Node2D root.
+    - Create a new 2D scene with a Node2D root.
     - Right click the node and rename it to `MainGame`

```

Comment:
Since we'll have a `Main` scene later on in the tutorial, should we call this scene `Demo` or `ArguingTrolls` to avoid confusion? Just an idea. I don't know if there is a right/conventional name to use. (If changed, would also need to change the name of `MainGame.cs`.)

Reply:
Renamed to ArguingTrolls